routes/users.js

```diff
@@ -1,8 +1,9 @@
 const express = require('express');
-const { createUser } = require('../controllers/userC');
+const { createUser, userLogin } = require('../controllers/userC');
 
 const userRoute = express.Router();
 
 userRoute.post('/register', (createUser));
+userRoute.post('/userLogin', (userLogin));
```

Comment:
```suggestion
userRoute.post('/login', (userLogin));
```

Update this route

Reply:
Okay, noted

Reply:
Corrections has been made and tested so its good to go.